app/javascript/shared/components/MarkdownEditor.res

```diff
@@ -17,6 +17,8 @@ type rec selection = (selectionStart, selectionEnd)
 and selectionStart = int
 and selectionEnd = int
 
+type currentFile = string
```

Comment:
Let's have this changed to 
```rescript
type currentFileName = option<string>
```

Just for better naming and an option type instead,